app/services/ping.rb

```diff
@@ -0,0 +1,100 @@
+class Ping
+  STATUS_OK = 'ok'.freeze
+  STATUS_FAIL = 'FAIL'.freeze
```

Comment:
Oh here are the constants. OK.

Reply:
@lzap, thanks for review :) If no one has concerns I'll use those constants in core and change the Katello PR to you them.